components/generators/kubernetes/__init__.py

```diff
@@ -632,7 +632,7 @@ def add_volume_mounts_from_configs(self):
         secrets = container.secrets.items()
         for object_name, spec in configs:
             if spec is None:
-                raise CompileError(
+                raise ABORT_EXCEPTION_TYPE(
```

Comment:
You don't need to use `ABORT_EXCEPTION_TYPE`, this is already set to `CompileError` in the kadet input type